packages/@expo/cli/src/export/embed/resolveOptions.ts

```diff
@@ -0,0 +1,78 @@
+import arg from 'arg';
+import path from 'path';
+
+import { env } from '../../utils/env';
+import { CommandError } from '../../utils/errors';
+import { resolveCustomBooleanArgsAsync } from '../../utils/resolveArgs';
+
+export interface Options {
+  assetsDest?: string;
+  entryFile: string;
+  resetCache: boolean;
+  resetGlobalCache: boolean;
+  transformer?: string;
+  minify?: boolean;
+  config?: string;
+  platform: string;
+  dev: boolean;
+  bundleOutput: string;
+  bundleEncoding?: string;
+  maxWorkers?: number;
+  sourcemapOutput?: string;
+  sourcemapSourcesRoot?: string;
+  sourcemapUseAbsolutePath: boolean;
+  verbose: boolean;
+  unstableTransformProfile?: string;
+  generateStaticViewConfigs: boolean;
+}
+
+function isBool(val: any): asserts val is boolean {
```

Comment:
Make clear that this function performs an assertion, otherwise it sounds like a predicate function that returns true or false:
```suggestion
function assertIsBoolean(val: any): asserts val is boolean {
```